app/assets/stylesheets/responsive.scss

```diff
@@ -1,6 +1,7 @@
 /* Bootstrap overrides go here. This is included after bootstrap. */
 $navCollapseBackgroundColor: #f7f7f7;
 $navCollapseColor: #777;
+$navbarActiveBorderColor: #aaa;
```

Comment:
Do we need this? It looks like this variable is already declared in `nucore-variables.scss` (although, I forget the load order, so this might be needed).